VENDORS.md

```diff
@@ -24,67 +25,64 @@ If you notice any mistakes, or would like to contribute, feel free to **make** a
 | [415keys](https://415keys.com)                               | [x]      | [x]  | [x]   |        |         |           | Richmond, NC               |
-| [Alphakeys (US)](https://alphakeys.com/)                     | [x]      |      |       |        |         |           | ?                          |
-| [Asceny](https://asceny.com/)                                |          |      |       | [x]    |         |           | ?                          |
-| [BoardSource](https://boardsource.xyz/)<sup>1</sup>          |          |      |       |        |         | [x]       | ?                          |
+| [Alphakeys (US)](https://alphakeys.com/)                     | [x]      | [x]  | [x]   |        |         |           | ?                          |
+| [Asceny](https://asceny.com/)                                |          |      | [x]   | [x]    |         | [x]       | ?                          |
+| [BoardSource](https://boardsource.xyz/)<sup>1</sup>          | [x]      | [x]  | [x]   |        | [x]     | [x]       | ?                          |
 | [Cable Craze](https://cablecraze.net/)                       |          |      |       | [x]    |         |           | ?                          |
 | [Cannon Keys](https://cannonkeys.com/)                       | [x]      | [x]  | [x]   | [x]    | [x]     | [x]       | Rhode Island               |
-| [ClueBoard](https://clueboard.co/)                           | [x]      | [x]  | [x]   |        | [x]     | [x]       | ?                          |
 | [Checkerboards](https://www.checkerboards.xyz)<sup>1</sup>   | [x]      |      | [x]   |        |         | [x]       | Waco, TX
 | [Coaxius](https://coaxius.com/)                              |          |      |       | [x]    |         |           | California                 |
 | [Cookie Cables](https://cookiecables.com/)                   |          |      |       | [x]    |         |           | ?                          |
 | [Cruz Ctrl](https://cruzctrl.gg/)                            |          |      |       | [x]    |         |           | Bay Area, CA               |
 | [Cup Cables](https://cupcables.com/)                         |          |      |       | [x]    |         |           | Oakland, CA                |
-| [DangKeebs](https://dangkeebs.com/)                          | [x]      | [x]  |       |        |         |           | Boston, MA                 |
-| [Digital Carpentry](https://digital-carpentry.com/)          |          | [x]  |       |        |         |           | Kentucky                   |
+| [DangKeebs](https://dangkeebs.com/)                          | [x]      | [x]  | [x]   |        | [x]     |           | Las Vegas, NV              |
+| [Digital Carpentry](https://digital-carpentry.com/)          |          |      |       |        |         |           | Kentucky                   |
```

Comment:
only thing left i can see from them is a macropad and maybe tools, how do we wanna treat this

Reply:
they do have some niche accessories like display stands so I'd say keep it?